zcfan.c

```diff
@@ -202,11 +202,11 @@ static void maybe_ping_watchdog(void) {
 }
 
 #define CONFIG_PATH "/etc/zcfan.conf"
-#define fscanf_threshold(f, pos, name, fl)                                     \
+#define fscanf_try(f, pos, name, fl)                                           \
```

Comment:
`fscanf_try` doesn't say what it's trying to scan, like `fscanf_threshold` did before. It tries to scan an `int` next to a key, so it should be something like `fscanf_int_for_key` since it's made generic.

Reply:
Done.